src/quacc/recipes/emt/defects.py

```diff
@@ -0,0 +1,98 @@
+"""Defect recipes for EMT"""
```

Comment:
FYI: I formatted your code with `isort .` and then `black .` per the style quacc uses, in case you're curious. Most packages have a strict requirement to match their formatting, but I don't care --- it'll get reformatted on merge. Just letting you know though.